node/libs/protobuf_build/src/canonical.rs

```diff
@@ -1,6 +1,7 @@
 //! Checks whether messages in the given file descriptor set support canonical encoding.
 use crate::syntax::extract_message_names;
 use anyhow::Context as _;
+use protox::prost_reflect::{self, prost_types};
```

Comment:
If we depend on reexports, let's do this consistently. protobuf won't compile if protox::prost_reflect is different than prost_reflect (right?), but if we don't want to import everywhere protox (only in the build scripts) then lets import prost_reflect directly.